src/c++/perf_analyzer/client_backend/client_backend.cc

```diff
@@ -119,13 +119,14 @@ ClientBackendFactory::Create(
     std::shared_ptr<Headers> http_headers,
     const std::string& triton_server_path,
     const std::string& model_repository_path, const bool verbose,
-    const std::string& metrics_url,
+    const std::string& metrics_url, const cb::ContentType input_content_type,
+    const cb::ContentType output_content_type,
     std::shared_ptr<ClientBackendFactory>* factory)
 {
   factory->reset(new ClientBackendFactory(
       kind, url, protocol, ssl_options, trace_options, compression_algorithm,
       http_headers, triton_server_path, model_repository_path, verbose,
-      metrics_url));
+      metrics_url, input_content_type, output_content_type));
```

Comment:
recommend not using content_type - but something like input_data_format (to avoid the overloaded term with http content types)

Reply:
I think @Tabrizian suggested keeping content type. If that's so, can y'all come to an agreement? I'm fine either way.

Reply:
I'm fine either way.

Reply:
@matthewkotila dealer's choice :)

Reply:
I think I lean towards using content type because it does actually cause the http request content type to change if all inputs are in json format:

https://github.com/triton-inference-server/client/blob/matthewkotila-json-inputs-outputs/src/c%2B%2B/library/http_client.cc#L2156-L2160

Reply:
I'm also concerned calling it something like input data format will confuse users with regard to the input datatype that they set in the `--input-data=file.json` file and the model config.

Reply:
Maybe input_tensor_format ? Last though on the matter :) Just looking to see how to keep it similar to the terminology:

https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/required_api.md#tensor-data

https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_binary_data.md

Reply:
I like input tensor format